pkg/model/storage/storage.go

```diff
@@ -75,16 +77,18 @@ type Storage struct {
 	Events *packageEvents
 }
 
-func New(databaseDirectory string, store kvstore.KVStore, cachesProfile *profile.Caches) *Storage {
+func New(databaseDirectory string, store kvstore.KVStore, cachesProfile *profile.Caches, belowMaxDepth int) *Storage {
 
 	utxoManager := utxo.New(store)
 
 	s := &Storage{
-		databaseDir: databaseDirectory,
-		store:       store,
-		utxoManager: utxoManager,
+		databaseDir:   databaseDirectory,
+		store:         store,
+		utxoManager:   utxoManager,
+		belowMaxDepth: milestone.Index(belowMaxDepth),
```

Comment:
why is this a milestone.Index?

Reply:
all the calculations we do with belowMaxDepth are based on milestones. So I don't need to convert later on.